File: app/src/main/java/com/jetsynthesys/rightlife/ui/payment/ColorCode.java
package com.jetsynthesys.rightlife.ui.payment;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ColorCode {

    @SerializedName("card")
    @Expose
    private String card;
    @SerializedName("promo")
    @Expose
    private String promo;
    @SerializedName("item")
    @Expose
    private String item;

    public String getCard() {
        return card;
    }

    public void setCard(String card) {
        this.card = card;
    }

    public String getPromo() {
        return promo;
    }

    public void setPromo(String promo) {
        this.promo = promo;
    }

    public String getItem() {
        return item;
    }

    public void setItem(String item) {
        this.item = item;
    }

}

File: app/src/main/java/com/jetsynthesys/rightlife/ui/settings/pojo/ColorCode.kt
package com.jetsynthesys.rightlife.ui.settings.pojo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ColorCode {
    @SerializedName("card")
    @Expose
    var card: String? = null

    @SerializedName("promo")
    @Expose
    var promo: String? = null

    @SerializedName("item")
    @Expose
    var item: String? = null
}